DB/src/main/java/io/deephaven/db/util/PythonScopeJpyImpl.java

```diff
@@ -139,12 +139,16 @@ private static CallableWrapper parseNumbaVectorized(PyObject pyObject, String nu
         char numpyTypeCode = numbaFuncTypes.charAt(numbaFuncTypes.length() - 1);
         Class returnType = numpyType2JavaClass.get(numpyTypeCode);
         if (returnType == null) {
-            throw new IllegalArgumentException("numba vectorized functions must have a return type.");
+            throw new IllegalArgumentException("numba vectorized functions must have a return type in integers, floating points, or boolean.");
```

Comment:
I prefer this wording

"must have an integral, floating point, or boolean return type"